modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -360,7 +360,7 @@ $ docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig registr
 
 You can run only the latency test using the `-ginkgo.focus` parameter.
```

Comment:
Seem to be stating this twice - why ? We have updated other commands later to read podman can we update this command and others to use podman rather than docker?